src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypemicro.py

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2022 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for the pypemicro module: https://github.com/nxpmicro/pypemicro
+
+import glob
+import os
+from PyInstaller import isolated
+from PyInstaller.utils.hooks import remove_prefix, get_package_paths
+
+
+@isolated.decorate
+def get_safe_libs():
+    from pypemicro import PyPemicro
+    libs = PyPemicro.get_pemicro_lib_list()
+    return libs
+
+
+pkg_base, pkg_dir = get_package_paths("pypemicro")
+
+binaries = list()
+for lib in get_safe_libs():
+    source = lib['path']
+    for file in glob.glob(os.path.join(source, '*')):
```

Comment:
Why do we need to glob the library directory? Shouldn't we just be collecting the `os.path.join(lib['path'], lib['name'])` instead? Otherwise, we'll be collecting both 32-bit and 64-bit library wheras only one of them is really required. 

I see that macOS also ships `libusb.dylib`, but I see no indication that it is used anywhere (it is not link-time dependency of `unitacmp-64.dylib` nor is it loaded dynamically by the python code. Might be dynamically loaded by some function in `unitacmp-64.dylib`, though?)

Reply:
> I see that macOS also ships `libusb.dylib`, but I see no indication that it is used anywhere (it is not link-time dependency of `unitacmp-64.dylib` nor is it loaded dynamically by the python code. Might be dynamically loaded by some function in `unitacmp-64.dylib`, though?)

Hmm, the strings in `unitacmp-64.dylib` seem to imply that `libusb.dylib` might be dynamically loaded somewhere. Still, I think it might be better to collect just `os.path.join(lib['path'], lib['name'])` on all OSes, and in addition collect `os.path.join(lib['path'], 'libusb.dylib')` on macOS.